### ISSUE TITLE
Collaboration with the Ibis Team

### ISSUE THREAD

**Issue by cpcloud** [snowflakedb/snowpark-python#759]
Hello Snowpark team 👋🏻!

I'm Phillip Cloud, the lead developer of [the Ibis Project](https://github.com/ibis-project/ibis).

Snowpark has been on our radar as a library to watch for potential collaborations, and I would like to kick that off.

It seems like we may have gotten off on the wrong foot somewhere, but I would like to extend an olive branch and invite y'all to comment on [this ibis issue](https://github.com/ibis-project/ibis/issues/5877) coming from a user's request to add a Snowpark backend to ibis.

To summarize: we have some concerns but would love to put in the effort to have a tight collaboration.

We have a few (hopefully reasonable!) requests from y'all listed in the issue.

Let's get this rocket ship off the ground 🚀!

**Comment by sfc-gh-yixie** [comment]
@cpcloud Thanks for reaching out. I've rendered the message to the team. Will update you.
